src/snap/getAllNotifications.ts

```diff
@@ -0,0 +1,17 @@
+import { Page } from "puppeteer";
+import { clickOnElement, openProfileDropdown } from "../helpers";
+import { NotificationList } from "./types";
+
+export const getAllNotifications =
+  (page: Page) => async (): Promise<NotificationList> => {
+    await page.bringToFront();
+    await openProfileDropdown(page);
+    await clickOnElement(page, "Notifications");
+    await page.waitForTimeout(300);
+
+    return await page.$$eval(
```

Comment:
after we get notifications we should probably head back to home screen before returning